Source/PagesController.swift

```diff
@@ -40,10 +40,11 @@ extension PagesController {
 
   public func goto(index: Int) {
     if index > -1 && index < pages.count {
+      let direction: UIPageViewControllerNavigationDirection = (index > currentIndex) ? .Forward : .Reverse
```

Comment:
Do we need to add type annotation `UIPageViewControllerNavigationDirection` to be able to use `.Forward` and `.Reverse`, right?